.npmrc

```diff
@@ -0,0 +1,5 @@
+registry=https://registry.yarnpkg.com/
+  
+@internxt:registry=https://npm.pkg.github.com
+//npm.pkg.github.com/:_authToken=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
```

Comment:
this is astonishing, add this to the .gitignore and disable this token ASAP

Reply:
I don't understand why this file was not in gitignore.